client/app/dbaas/dbaas-metrics/header/metrics-header.html

```diff
@@ -1,15 +1,44 @@
-                 data-state="dbaas.metrics.detail.token" 
-                 data-state-params="{ serviceName: MetricsHeaderCtrl.serviceName }"></cui-tab>
-        <cui-tab data-text="'metrics_tab_platform' | translate" 
-                 data-state="dbaas.metrics.detail.platform" 
-                 data-state-params="{ serviceName: MetricsHeaderCtrl.serviceName }"></cui-tab>
-    </cui-tabs>
-</cui-page-header>
\ No newline at end of file
+<header class="oui-page-header">
+    <div class="oui-page-header__container">
+        <div class="oui-page-header__content">
+            <oui-spinner data-size="m" data-ng-if="MetricsDetailCtrl.loadingService"></oui-spinner>
+            <h1 class="oui-page-header__title" data-ng-if="!MetricsDetailCtrl.loadingService" data-ng-bind="MetricsDetailCtrl.service.description || MetricsDetailCtrl.serviceName"></h1>
+            <h2 class="oui-page-header__description" data-ng-if="!MetricsDetailCtrl.loadingService" data-ng-bind="MetricsDetailCtrl.service.description === MetricsDetailCtrl.serviceName ? '' : MetricsDetailCtrl.serviceName"></h2>
+        </div>
+        <oui-guide-menu data-text="{{ MetricsHeaderCtrl.guides.title }}">
+            <oui-guide-menu-group 
+                data-ng-repeat="section in MetricsHeaderCtrl.guides.sections track by section.title"
+                data-label="{{section.title}}">
+                <oui-guide-menu-item
+                    data-ng-repeat="guideItem in section.list track by guideItem.url"
+                    data-text="{{ guideItem.name }}"
+                    data-href="{{ guideItem.url }}"
+                    data-external="guideItem.external">
+                </oui-guide-menu-item>
+            </oui-guide-menu-group>
+            <oui-guide-menu-divider></oui-guide-menu-divider>
+            <oui-guide-menu-item
+                data-text="{{ MetricsHeaderCtrl.guides.footer.name }}"
+                data-href="{{ MetricsHeaderCtrl.guides.footer.url }}"
+                data-external="MetricsHeaderCtrl.guides.footer.external">
+            </oui-guide-menu-item>
+        </oui-guide-menu>
+    </div>
+    <oui-header-tabs>
+        <oui-header-tabs-item
+            data-text="{{ ::'metrics_tab_dashboard' | translate }}"
+            data-state="dbaas.metrics.detail.dashboard"
+            data-state-params="{ serviceName: MetricsHeaderCtrl.serviceName }">
+        </oui-header-tabs-item>
+        <oui-header-tabs-item
+            data-text="{{ ::'metrics_tab_token' | translate }}" 
+            data-state="dbaas.metrics.detail.token" 
+            data-state-params="{ serviceName: MetricsHeaderCtrl.serviceName }">
+        </oui-header-tabs-item>
+        <oui-header-tabs-item
+            data-text="{{ ::'metrics_tab_platform' | translate }}" 
+            data-state="dbaas.metrics.detail.platform"
+            data-state-params="{ serviceName: MetricsHeaderCtrl.serviceName }">
+        </oui-header-tabs-item>
+    </oui-header-tabs>
+</header>
```

Comment:
Newline needed

Reply:
I have added new line but its not reflecting here

Reply:
Yes sorry, it is not display :)